var/spack/repos/builtin/packages/git/package.py

```diff
@@ -305,3 +305,14 @@ def install_manpages(self):
             install_tree('man1', prefix.share.man.man1)
             install_tree('man5', prefix.share.man.man5)
             install_tree('man7', prefix.share.man.man7)
+
+    def setup_run_environment(self, env):
+        # Setup run environment if using SVN extension
+        # Libs from perl-alien-svn and apr-util are required in
+        # LD_LIBRARY_PATH
+        # TODO: extend to other platforms
+        if "+svn" in self.spec and sys.platform.startswith('linux'):
+            perl_svn = self.spec['perl-alien-svn']
+            env.prepend_path('LD_LIBRARY_PATH', join_path(
```

Comment:
If you want to extend this to macOS, you need to set `DYLD_FALLBACK_LIBRARY_PATH` instead.

Reply:
Ok thanks, I'll make a follow on PR at some point when I can spend some more time on macOS.